client/packages/common/src/types/schema.ts

```diff
@@ -4822,7 +4823,7 @@ export enum StocktakeLineSortFieldInput {
   ExpiryDate = 'expiryDate',
   ItemCode = 'itemCode',
   ItemName = 'itemName',
-  LocationName = 'locationName',
+  LocationCode = 'locationCode',
```

Comment:
this was a general request, that we use location code not name. took the opportunity to update.

Reply:
Louisa was the instigator and she was backed up by others who said that code is used in preference to name. the name is just too long to show in tables.
I took the YAGNI principle - we aren't planning to use name as a column or sort option, so I haven't implemented.